### DISCUSSION TITLE
Accessing last message outside of current flow session

### DISCUSSION THREAD

**Discussion by LeahBreedenNyaruka** [nyaruka/rapidpro#4712] [Ideas]
`@input` will only reference the last input from the current session. User requested a way to easily access the last input from a contact even after completing a run.

**Comment by ericnewcomer** [comment]
As in in the first step of a new flow session without user input? This sounds pretty edge-casey.

**Reply by LeahBreedenNyaruka** [reply]
Eran asks, I add it 😂
